docs/resources/artifactory_anonymous_user.md

```diff
@@ -0,0 +1,27 @@
+# Artifactory Anonymous User Resource
+
+Provides an Artifactory anonymous user resource. This can be used to import Artifactory user.
+
+Note: Anonymous user cannot be altered once imported into Terraform state.
+
+## Example Usage
+
+```hcl
+# Create a new Artifactory user called terraform
+resource "artifactory_anonymous_user" "anonymous-user" {
```

Comment:
You can't create an anonymous user, you can only import

Reply:
Good catch, I forgot to change that part in the doc.